wingui/pdcscrn.c

```diff
@@ -1,7 +1,9 @@
 /* Public Domain Curses */
 
 #include "pdcwin.h"
+#include <shellapi.h>
 #include <tchar.h>
+#include <stdlib.h>
 #include <stdint.h>
```

Comment:
Similar to the issue @GitMensch points out for `wincon/pdcscrn.c`.  Both `shellapi.h` and `stdlib.h` are already #included if `WIN32_LEAN_AND_MEAN` is defined.  Are you finding problems on older systems if they aren't always included?

If so,  your patch should also remove the (now duplicate) #including of both files when `WIN32_LEAN_AND_MEAN` is defined.

Reply:
includes are ok now